pkgs/tools/security/ronin/default.nix

```diff
@@ -15,6 +15,14 @@ bundlerEnv {
     };
   };
 
+  postBuild = ''
+    for i in $out/bin/*; do
+      if [[ "$i" != *"/ronin"* ]]; then
```

Comment:
Isn't this just `find $out/bin -not -name "ronin*" -delete`? or am I missing something?

Reply:
Come to think of it - there's probably some bash extglob solution too, but I've never been particularly good at those...

Reply:
No, you're right. I was not good at using `find` command and didn't know about extglob.